macros/get_enabled_unioned_models.sql

```diff
@@ -0,0 +1,33 @@
+{% macro get_enabled_unioned_models() %}
```

Comment:
This is a pretty custom macro so I ultimately think it belongs to live only within this package.

Additionally, this macro is understandable to me, but it may not be to others down the line. Would you be able to [document this macro in a yml file](https://docs.getdbt.com/faqs/docs/documenting-macros) to provide a short concise description as to what is going on. In particular it would be helpful why we needed the additional conditionals for the bill payment and credit card payment. This will serve helpful down the road for maintainability.

Reply:
macro slightly refactored and yml file created! https://github.com/fivetran/dbt_quickbooks/blob/feature/general-ledger-unioning/macros/macros_docs.yml